clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -18901,6 +18901,160 @@ case Builtin::BI__builtin_hlsl_elementwise_isinf: {
         CGM.getHLSLRuntime().getRadiansIntrinsic(), ArrayRef<Value *>{Op0},
         nullptr, "hlsl.radians");
   }
+  // This should only be called when targeting DXIL
```

Comment:
This comment seems old and maybe should be moved to line 18921.  SPIRV  implementation is on 18952.